### PR TITLE
Support additional ino field seen on openSUSE 15.4 (to parse the prog_tag)

### PR DESCRIPTION
**Pull request by chwillso**
Without this fix (or one like it): one ends up getting this warning when any bpf program loads
```
WARNING: cannot get prog tag, ignore saving source with program tag 
prog tag mismatch 71d4822dd06c5b8e 1 
```
thus one can't get the source of the bpf programs output to /var/tmp/bcc

Seen on openSUSE 15.4 for bpf programs.

Other OSes cat /proc/self/fdinfo/*
```
pos:    0
flags:  02000002
mnt_id: 15
prog_type:      2
```
On openSUSE SP4 cat /proc/self/fdinfo/*
```
pos:    0
flags:  02000002
mnt_id: 15
ino:       11319
prog_type:      2
```
The additional field causes the prog_tag not to be retrieved properly.

### CODE REVIEW

File: src/cc/libbpf.c [hunk 1]
```diff
@@ -620,7 +620,11 @@ int bpf_prog_get_tag(int fd, unsigned long long *ptag)
   fgets(fmt, sizeof(fmt), f); // pos
   fgets(fmt, sizeof(fmt), f); // flags
   fgets(fmt, sizeof(fmt), f); // mnt_id
-  fgets(fmt, sizeof(fmt), f); // prog_type
+  fgets(fmt, sizeof(fmt), f); // prog_type or ino
```
Comment (chenhengqi) [inline src/cc/libbpf.c:623]: How about using sscanf here ? This will make the code more readable and reliable.

Comment (chwillso) [reply src/cc/libbpf.c:623]: Totally agree if that is considered an ok change and not too big, works for me.

Comment (chwillso) [reply src/cc/libbpf.c]: ec45eb957415c568a25853acfc25837c9bff891f

**Review by chenhengqi — COMMENTED** [review]
LGTM.

**Comment by yonghong-song** [comment]
@chwillso could you merge 3 commits to a single commit, rebase on top of master branch and resubmit?

**Comment by yonghong-song** [comment]
@chwillso could you merge 3 commits to a single commit, rebase on top of master branch and resubmit?

**Comment by chwillso** [comment]
> @chwillso could you merge 3 commits to a single commit, rebase on top of master branch and resubmit?
@yonghong-song hopefully  did the trick.

**Pull request merged**